backend/tests/integration/records/views/test_views_conferences.py

```diff
@@ -264,3 +266,63 @@ def test_date_range_contains_conferences_filter(api_client, db, es, create_recor
     assert 1 in found_recids
     assert 2 in found_recids
     assert 3 in found_recids
+
+
+@freeze_time(datetime(2019, 9, 15, 0, 0, 0))  # today is the 9th of September, 2019
```

Comment:
```suggestion
@freeze_time('2019-09-15')  # today is the 9th of September, 2019
```

Reply:
much easier to read :)

Reply:
was just following the codebase styles 😄